jetty-osgi/test-jetty-osgi/src/test/java/org/eclipse/jetty/osgi/test/TestJettyOSGiClasspathResources.java

```diff
@@ -70,8 +69,9 @@
             "com.sun.org.apache.xpath.internal.jaxp", "com.sun.org.apache.xpath.internal.objects"));
 
         options.addAll(TestOSGiUtil.coreJettyDependencies());
-        options.add(mavenBundle().groupId("biz.aQute.bnd").artifactId("bndlib").versionAsInProject().start());
-        options.add(mavenBundle().groupId("org.ops4j.pax.tinybundles").artifactId("tinybundles").version("2.1.1").start());
+        //Note: we have to back down the version of bnd used here because tinybundles expects only this version
+        options.add(mavenBundle().groupId("biz.aQute.bnd").artifactId("biz.aQute.bndlib").version("3.5.0").start());
```

Comment:
Shouldn't this be `.versionAsInProject()` instead of `.version("3.5.0")` ?

Reply:
Nope, the comment above is telling you why not: when using tinybundles we have to back down the version of bndlib for compatibility.